jenkins/osa_onmetal.functions.groovy

```diff
@@ -347,12 +347,15 @@ def start_api_uptime_tests() {
 
     String host_ip = get_onmetal_ip()
 
-    // run the API uptime tests
+    // run the API uptime tests.  Note that the output directory goes up 2
+    // levels.  That is done to ensure that the output file goes into the root
+    // for this project, rather than buried in the api_uptime/api_uptime
+    // subdirectory.
```

Comment:
It is more flexible in deed, but the problem I see is that when working with Jenkins you not only depend on the OS but you also depend on the workspace your job is using, every job uses a different workspace in different systems, and depending on the executioner that is running the job, so using relative paths can lead to jenkins running one part of the pipeline in one workspace and other part of the pipeline in another workspace (which is ok) which would lead the pipeline to break cause it may not find the other resource